api/envoy/extensions/matching/common_inputs/network/v3/network_inputs.proto

```diff
@@ -95,3 +96,27 @@ message TransportProtocolInput {
 // [#extension: envoy.matching.inputs.application_protocol]
 message ApplicationProtocolInput {
 }
+
+// Input that matches the authentication attributes for a downstream.
+// [#extension: envoy.matching.inputs.authenticated]
+message AuthenticatedInput {
```

Comment:
What is the reasoning behind grouping these together in the API? Is it motivated mostly by reducing the boilerplate on the implementation side? If I knew nothing about the implementation I think I would have expected these to be independent extensions

Reply:
I agree. It seems like it would make sense to have a different matcher for each of these.

Reply:
Do you mean we shall have different authenticated matchers (e.g, URI SAN matcher, DNS SAN matcher), or we shall separate authenticated input from network inputs?